get_session_request.php

```diff
@@ -0,0 +1,28 @@
+<?php
+require_once 'config.php';
+
+function get_session_request($contents) {
+    $parsedjson = json_decode($contents, true);
+
+    $randomnum = rand(1,9);
+    for($i=0; $i<10; $i++)
+        $randomnum .= rand(0,9);
+
+    $sessionrequest = [
+        '@context' => 'https://irma.app/ld/request/issuance/v2',
+        'credentials' => [[
+            'credential' => 'irma-demo.IRMATube.member',
```

Comment:
This issues an `irma-demo` credential while just before a `pbdf` credential was used. Because of the following two facts:

* When doing session `n` in a session chain, all attributes that were disclosed previously in session `n-1` and before are disclosed again automatically and invisibly during session `n`, to bind session `n` to the phone that did session `n-1`. So, this session request makes the IRMA server not only issue this `irma-demo` credential but also request a `pbdf` credential, simultaneously in the second session.
* The IRMA server currently fails on issuance sessions mixing `irma-demo` and `pbdf`.

The latter point is probably a bug in the IRMA server. Regardless, it would be better I think to not mix schemes in this fashion. I suggest that you:
* add a new boolean in `config.php` called `DEMO` or something;
* if that is enabled, verify in the first session `irma-demo.gemeente.personalData.fullname` and issue in the second session `irma-demo.IRMATube.member`, like you do now;
* if it is disabled, verify `pbdf.gemeente.personalData.fullname` and issue the production IRMATube credential, `pbdf.pbdf.irmatube`.